puncover/puncover.py

```diff
@@ -62,6 +73,14 @@ def main():
 
     if args.debug:
         app.debug = True
+
+    # Open a browser window, only if this is the first instance of the server
+    # from https://stackoverflow.com/a/63216793
+    if not args.no_open_browser and not os.environ.get("WERKZEUG_RUN_MAIN"):
```

Comment:
sorry for being late to the show, @noahp. Will this negatively affect existing users and their automation? e.g. will a CI system that's not strictly headless end up with a galore of open windows after a week?

Reply:
that said I'm quite happy to make this flag opt-in, LMK if you think that's a better way

Reply:
I think it's ok, thanks for checking. I don't have data for that but considering that puncover can no longer produce a report, I think that almost all users are using it interactively anyway.

One last question (I might get to answer myself one day): Will that call work on CDEs such as Gitpod?

Reply:
But you can also click the popup to open the tab for previewing, so it's not 100% terrible. Probably some gitpod-specific logic could be added to make it less awkward but it does mostly work.